### ISSUE TITLE
URL Pattern for VectorBase is invalid

### ISSUE THREAD

**Issue by piotr-gawron** [identifiers-org/identifiers-org.github.io#187]
Current URL pattern looks like: `https://www.vectorbase.org/search/site/{$id}?&site="Genome"` and because of that sample url cannot be resolved https://identifiers.org/vectorbase:ISCW007415.

I assume that quota characters are the problem. However, the url without extra quota characters goes to "Page not found" response. I think the proper patter should look like: `https://vectorbase.org/vectorbase/app/search?q={$id}`.

**Comment by cthoyt** [comment]
Hi @piotr-gawron, while you wait for the Identifiers.org team to look into this, you can check the Bioregistry project at https://bioregistry.io and https://github.com/biopragmatics/bioregistry, which has an open governance model that makes it easier to make updates. It also has a resolver that can be used a drop-in replacement for identifiers.org, so you can try using https://bioregistry.io/vectorbase:ISCW007415, which seems to link properly (we previously stumbled on the same issue as this and made an override)

**Comment by renatocjn** [comment]
Dear @piotr-gawron, thank you for catching this issue, are you part of the vectorbase team?

I have found these two links using the sample ID that I could turn into the new URL pattern:

https://vectorbase.org/vectorbase/app/record/gene/ISCW007415
Which seems preferable because it directly takes to the artifact information. But it may be too specific

and

https://vectorbase.org/vectorbase/app/search?q=ISCW007415
Which seems to search everything available at vectorbase but I'm afraid it can not be ideal since it may return more than one result for a query.

What do you think?

**Comment by piotr-gawron** [comment]
Dear @renatocjn

I'm a developer who uses your API for resolving some identifiers. At some point I decided to introduce unit test to verify if your API resolves proper pages with sample identifiers and I found some issues that I reported. I cannot help you with the external databases because I'm not familiar with them.

Nevertheless, in this example I think this one is better:  https://vectorbase.org/vectorbase/app/record/gene/ISCW007415
